src/app/backend/resource/thirdpartyresource/detail.go

```diff
@@ -41,14 +45,50 @@ func GetThirdPartyResourceDetail(client k8sClient.Interface, name string) (*Thir
 		return nil, err
 	}
 
-	return getThirdPartyResourceDetail(thirdPartyResource), nil
+	return getThirdPartyResourceDetail(config, thirdPartyResource), nil
 }
 
-func getThirdPartyResourceDetail(thirdPartyResource *extensions.ThirdPartyResource) *ThirdPartyResourceDetail {
+func getThirdPartyResourceDetail(config clientcmd.ClientConfig, thirdPartyResource *extensions.ThirdPartyResource) *ThirdPartyResourceDetail {
 	return &ThirdPartyResourceDetail{
 		ObjectMeta:  common.NewObjectMeta(thirdPartyResource.ObjectMeta),
 		TypeMeta:    common.NewTypeMeta(common.ResourceKindThirdPartyResource),
 		Description: thirdPartyResource.Description,
 		Versions:    thirdPartyResource.Versions,
+		Objects:     getThirdPartyResourceObjects(config, thirdPartyResource),
 	}
 }
+
+// TODO(maciaszczykm): It should always take preffered version. Is there any Kubernetes API to use instead of this function?
+func getThirdPartyResourceGroupVersion(thirdPartyResource *extensions.ThirdPartyResource) schema.GroupVersion {
+	version := ""
+	if len(thirdPartyResource.Versions) > 0 {
+		version = thirdPartyResource.Versions[0].Name
+	}
+
+	group := ""
+	if strings.Contains(thirdPartyResource.ObjectMeta.Name, ".") {
+		group = thirdPartyResource.ObjectMeta.Name[strings.Index(thirdPartyResource.ObjectMeta.Name, ".")+1:]
+	} else {
+		group = thirdPartyResource.ObjectMeta.Name
+	}
+
+	return schema.GroupVersion{
+		Group:   group,
+		Version: version,
+	}
+}
+
+// TODO(maciaszczykm): Is there any Kubernetes API to use instead of this function?
```

Comment:
AFAIR, no. But this issue has some interesting links you can dig into: https://github.com/kubernetes/kubernetes/issues/18622